spec/features/google_tag_manager_can_anonomise_page_url_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.feature 'Google Tag Manager' do
+  context 'When searching for jobs with home postcode filter' do
+    scenario 'parameters are removed from URL before sending URL to dataLayer', js: true do
+      expect(Geocoder).to receive(:coordinates).with('CH52DD')
+                                               .and_return([51.6622925, -0.1180655])
+
+      visit jobs_path
+
+      within '.filters-form' do
+        fill_in 'location', with: 'CH52DD'
+        select 'Within 5 miles'
+        page.find('.govuk-button[type=submit]').click
+      end
+
+      expect(evaluate_script('dataLayer')).to include(include('event' => 'parametersRemoved', 'dePIIedURL' => '/jobs'))
```

Comment:
I think this commit should also contain some more context as to why we're putting this URL into the `dataLayer`.

Reply:
Will amend the commit to contain more context